CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 2.2.5
+Thu 14 Jan 11:28:34 GMT 2021
+
+* [c829957e](https://github.com/hyperledger/fabric-sdk-node/commit/c829957e) [FABN-1686: Private data index parsing (release-2](https://jira.hyperledger.org/browse/FABN-1686: Private data index parsing (release-2).2) (#416)
```

Comment:
The changelog script seems to have been confused by this line and generated a garbled link. May need a manual tweak

Reply:
Changes made to brackets and dashes, missing links for jiras added